test/cli/validate/error/expected-stderr

```diff
@@ -1,16 +1,16 @@
 WARN	Error:
-WARN	- validation failed:
-WARN	  - config "foo/extractor/keboola.ex-azure-cost-management/with-schema/config.json" doesn't match schema:
+WARN	- Validation failed:
+WARN	  - Config "foo/extractor/keboola.ex-azure-cost-management/with-schema/config.json" doesn't match schema:
 WARN	    - missing properties: "subscriptionId"
-WARN	  - config row "foo/extractor/keboola.ex-azure-cost-management/with-schema/rows/row-1/config.json" doesn't match schema:
+WARN	  - Config row "foo/extractor/keboola.ex-azure-cost-management/with-schema/rows/row-1/config.json" doesn't match schema:
 WARN	    - "export": missing properties: "aggregation"
 WARN	    - "export.groupingDimensions": expected array, but got string
-WARN	  - config row "foo/extractor/keboola.ex-azure-cost-management/with-schema/rows/row-2/config.json" doesn't match schema:
+WARN	  - Config row "foo/extractor/keboola.ex-azure-cost-management/with-schema/rows/row-2/config.json" doesn't match schema:
 WARN	    - "export.type": value must be one of "ActualCost", "AmortizedCost", "Usage"
-WARN	  - config row "foo/extractor/keboola.ex-azure-cost-management/with-schema/rows/row-3/config.json" doesn't match schema:
+WARN	  - Config row "foo/extractor/keboola.ex-azure-cost-management/with-schema/rows/row-3/config.json" doesn't match schema:
 WARN	    - "export.type": value must be one of "ActualCost", "AmortizedCost", "Usage"
-WARN	  - config "foo/extractor/ex-generic-v2/testUnencrypted/config.json" contains unencrypted values:
+WARN	  - Config "foo/extractor/ex-generic-v2/testUnencrypted/config.json" contains unencrypted values:
 WARN	    - #unencrypedValue
 WARN	    - parameters.tokens[1].#value
```

Comment:
Rovnako aj dalsie specialne chyby, napr. unencrypted values, ... tu by to velke pismeno robilo problem.